_clients/go.md

```diff
@@ -134,10 +143,11 @@ blk, err := client.Bulk(
 `),
 	)
 ```
+{% include copy.html %}
 
 ## Searching for documents
 
-The easiest way to search for documents is to construct a query string. The following code uses a `multi_match` query to search for "miller" in the title and director fields. It boosts the documents where "miller" is in the title field. 
+The easiest way to search for documents is to construct a query string. The following code uses a `multi_match` query to search for "miller" in the title and director fields. It boosts the documents where "miller" is in the title field:
```

Comment:
Does the element following this require a brief introduction?

Reply:
This IS the introduction :)